### PR TITLE
Pioneer 20c gliders

### PR DESCRIPTION
**Pull request by cdobs**

### CODE REVIEW

**Comment by cdobs** [comment]
Quick look: 3210-02010
559: 3304-00150-00173
380: 3304-00150-00174
339: 3304-00150-00175
564: 3304-00180-00011
Note: A new FCP for 564 was not performed. This doc # is for the previous deployment (P20b).
The SUNA was swapped to SN 728 and it was re-deployed.

File: deployment/CP15MOAS-GL339_Deploy.csv [hunk 1]
```diff
@@ -0,0 +1,7 @@
...
+KH20240821,,,,CP15MOAS-GL339-00-ENG000000,1,1,2024-08-21T15:07:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00339,CGCON-GLDRCP-00339,35.94715,-74.82825,,200,200,CS,,
+KH20240821,,,,CP15MOAS-GL339-01-ADCPAM000,1,1,2024-08-21T15:07:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00339,CGINS-ADCPAM-49697,35.94715,-74.82825,,200,200,CS,,
+KH20240821,,,,CP15MOAS-GL339-02-FLORTM000,1,1,2024-08-21T15:07:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00339,CGINS-FLORTM-03185,35.94715,-74.82825,,200,200,CS,,
+KH20240821,,,,CP15MOAS-GL339-03-CTDGVM000,1,1,2024-08-21T15:07:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00339,CGINS-CTDGVM-00073,35.94715,-74.82825,,200,200,CS,,
+KH20240821,,,,CP15MOAS-GL339-04-DOSTAM000,1,1,2024-08-21T15:07:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00339,CGINS-DOSTAM-00543,35.94715,-74.82825,,200,200,CS; top mount,,
+KH20240821,,,,CP15MOAS-GL339-05-PARADM000,1,1,2024-08-21T15:07:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00339,CGINS-PARADM-50244,35.94715,-74.82825,,200,200,CS,,
```
Comment (cooleyky) [inline deployment/CP15MOAS-GL339_Deploy.csv:7]: looks good

File: deployment/CP15MOAS-GL379_Deploy.csv [hunk 2]
```diff
@@ -11,9 +11,9 @@ KH20240523,,KH20240609,,CP15MOAS-GL379-02-FLORTM000,2,1,2024-05-23T13:32:00,2024
...
\ No newline at end of file
+PH20240630,,KH20240821,,CP15MOAS-GL379-00-ENG000000,3,1,2024-06-30T13:08:00,2024-08-21T16:43:00,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00379,CGCON-GLDRCP-00379,35.95345,-74.76803,,1000,1000,OM,,
+PH20240630,,KH20240821,,CP15MOAS-GL379-01-ADCPAM000,3,1,2024-06-30T13:08:00,2024-08-21T16:43:00,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00379,CGINS-ADCPAM-49016,35.95345,-74.76803,,1000,1000,OM,,
+PH20240630,,KH20240821,,CP15MOAS-GL379-02-FLORTM000,3,1,2024-06-30T13:08:00,2024-08-21T16:43:00,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00379,CGINS-FLORTM-04194,35.95345,-74.76803,,1000,1000,OM,,
+PH20240630,,KH20240821,,CP15MOAS-GL379-03-CTDGVM000,3,1,2024-06-30T13:08:00,2024-08-21T16:43:00,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00379,CGINS-CTDGVM-09030,35.95345,-74.76803,,1000,1000,OM,,
+PH20240630,,KH20240821,,CP15MOAS-GL379-04-DOSTAM000,3,1,2024-06-30T13:08:00,2024-08-21T16:43:00,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00379,CGINS-DOSTAM-00496,35.95345,-74.76803,,1000,1000,OM; top mount,,
+PH20240630,,KH20240821,,CP15MOAS-GL379-05-PARADM000,3,1,2024-06-30T13:08:00,2024-08-21T16:43:00,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00379,CGINS-PARADM-50220,35.95345,-74.76803,,1000,1000,OM,,
```
Comment (cooleyky) [inline deployment/CP15MOAS-GL379_Deploy.csv:19]: looks good

File: deployment/CP15MOAS-GL380_Deploy.csv [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+KH20240821,,,,CP15MOAS-GL380-00-ENG000000,1,1,2024-08-21T14:04:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00380,CGCON-GLDRCP-00380,35.94527,-74.82335,,350,350,MA,,
+KH20240821,,,,CP15MOAS-GL380-01-ADCPAM000,1,1,2024-08-21T14:04:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00380,CGINS-ADCPAM-49696,35.94527,-74.82335,,350,350,MA,,
+KH20240821,,,,CP15MOAS-GL380-02-FLORTM000,1,1,2024-08-21T14:04:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00380,CGINS-FLORTM-04472,35.94527,-74.82335,,350,350,MA,,
+KH20240821,,,,CP15MOAS-GL380-03-CTDGVM000,1,1,2024-08-21T14:04:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00380,CGINS-CTDGVM-09338,35.94527,-74.82335,,350,350,MA,,
+KH20240821,,,,CP15MOAS-GL380-04-DOSTAM000,1,1,2024-08-21T14:04:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00380,CGINS-DOSTAM-00499,35.94527,-74.82335,,350,350,MA; top mount,,
+KH20240821,,,,CP15MOAS-GL380-05-PARADM000,1,1,2024-08-21T14:04:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00380,CGINS-PARADM-50242,35.94527,-74.82335,,350,350,MA,,
```
Comment (cooleyky) [inline deployment/CP15MOAS-GL380_Deploy.csv:7]: looks good

File: deployment/CP15MOAS-GL559_Deploy.csv [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+KH20240821,,,,CP15MOAS-GL559-00-ENG000000,1,1,2024-08-21T16:14:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00559,CGCON-GLDRCP-00559,35.94267,-74.77592,,1000,1000,OM,,
+KH20240821,,,,CP15MOAS-GL559-01-ADCPAM000,1,1,2024-08-21T16:14:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00559,CGINS-ADCPAM-72777,35.94267,-74.77592,,1000,1000,OM,,
+KH20240821,,,,CP15MOAS-GL559-02-FLORTM000,1,1,2024-08-21T16:14:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00559,CGINS-FLORTM-03190,35.94267,-74.77592,,1000,1000,OM,,
+KH20240821,,,,CP15MOAS-GL559-03-CTDGVM000,1,1,2024-08-21T16:14:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00559,CGINS-CTDGVM-09083,35.94267,-74.77592,,1000,1000,OM,,
+KH20240821,,,,CP15MOAS-GL559-04-DOSTAM000,1,1,2024-08-21T16:14:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00559,CGINS-DOSTAM-00539,35.94267,-74.77592,,1000,1000,OM; top mount,,
+KH20240821,,,,CP15MOAS-GL559-05-PARADM000,1,1,2024-08-21T16:14:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00559,CGINS-PARADM-50203,35.94267,-74.77592,,1000,1000,OM,,
```
Comment (cooleyky) [inline deployment/CP15MOAS-GL559_Deploy.csv:7]: looks good

File: deployment/CP15MOAS-GL339_Deploy.csv (hunk 1 above)
Comment (rtravis-whoi) [inline deployment/CP15MOAS-GL339_Deploy.csv:7]: looks good

File: deployment/CP15MOAS-GL379_Deploy.csv (hunk 2 above)
Comment (rtravis-whoi) [inline deployment/CP15MOAS-GL379_Deploy.csv:19]: looks good

File: deployment/CP15MOAS-GL380_Deploy.csv (hunk 3 above)
Comment (rtravis-whoi) [inline deployment/CP15MOAS-GL380_Deploy.csv:7]: looks good

File: deployment/CP15MOAS-GL559_Deploy.csv (hunk 4 above)
Comment (rtravis-whoi) [inline deployment/CP15MOAS-GL559_Deploy.csv:7]: looks good

File: deployment/CP15MOAS-PG564_Deploy.csv [hunk 5]
```diff
@@ -1,8 +1,15 @@
...
+KH20240821,,,,CP15MOAS-PG564-00-ENG000000,2,1,2024-08-22T13:39:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00564,CGCON-GLDRCP-00564,35.95823,-74.71948,,1000,1000,OF,,
+KH20240821,,,,CP15MOAS-PG564-01-CTDGVM000,2,1,2024-08-22T13:39:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00564,CGINS-CTDGVM-09307,35.95823,-74.71948,,1000,1000,OF,,
+KH20240821,,,,CP15MOAS-PG564-02-DOSTAM000,2,1,2024-08-22T13:39:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00564,CGINS-DOSTAM-00604,35.95823,-74.71948,,1000,1000,OF; top mount,,
+KH20240821,,,,CP15MOAS-PG564-03-FLORTM000,2,1,2024-08-22T13:39:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00564,CGINS-FLORTM-04093,35.95823,-74.71948,,1000,1000,OF,,
+KH20240821,,,,CP15MOAS-PG564-04-FLORTO000,2,1,2024-08-22T13:39:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00564,CGINS-FLORTO-01348,35.95823,-74.71948,,1000,1000,OF,,
+KH20240821,,,,CP15MOAS-PG564-05-NUTNRM000,2,1,2024-08-22T13:39:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00564,CGINS-NUTNRM-00728,35.95823,-74.71948,,1000,1000,OF,,
+KH20240821,,,,CP15MOAS-PG564-06-PARADM000,2,1,2024-08-22T13:39:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00564,CGINS-PARADM-50218,35.95823,-74.71948,,1000,1000,OF,,
```
Comment (rtravis-whoi) [inline deployment/CP15MOAS-PG564_Deploy.csv:15]: looks good

**Review by rtravis-whoi — APPROVED** [review]

File: deployment/CP15MOAS-PG564_Deploy.csv (hunk 5 above)
Comment (cooleyky) [inline deployment/CP15MOAS-PG564_Deploy.csv:15]: looks good

**Review by cooleyky — APPROVED** [review]
looks good

**Pull request merged**